CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+### 2.24.0
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.9.3](https://fluentbit.io/announcements/v1.9.3/)
+* Amazon CloudWatch Logs for Fluent Bit 1.7.0
+* Amazon Kinesis Streams for Fluent Bit 1.9.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Compared to `2.23.4` this release adds:
+* Bug - Resolve IMDSv1 fallback error introduced in 2.21.0 [aws-for-fluent-bit:259](https://github.com/aws/aws-for-fluent-bit/issues/259)
+* Bug - Cloudwatch Fix integer overflow on 32 bit systems when converting tv_sec to millis [fluentbit:3640](https://github.com/fluent/fluent-bit/issues/3640)
+* Enhancement - Only create Cloudwatch Logs log group if it does not already exist to prevent throttling [fluentbit:4826](https://github.com/fluent/fluent-bit/pull/4826)
+* Enhancement - Implement multiline buffered mode for docker log driver partial message support [fluentbit:4671](https://github.com/fluent/fluent-bit/pull/4671)
```

Comment:
the buffered mode already existed in 1.8 series, just say implemented docker log driver partial message support

Reply:
OK, done.

Not mentioning the priority event loop, since it's not aws specific.